exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -109,12 +111,11 @@
    * in reverse order, starting from the {@code heightMax}.
    * @param count Number of blocks to return.
    *        It should be in range [1, {@linkplain ExonumApi#MAX_BLOCKS_PER_REQUEST}]
-   * @param skipEmpty if {@code true}, then only non-empty blocks will be returned
+   * @param blockFilter blocks filtering option identifies to skip or include empty blocks
```

Comment:
@param blockFilter controls whether to skip blocks with no transactions / empty blocks?